src/Todo.js

```diff
@@ -1,7 +1,65 @@
 'use strict';
 
 class Todo {
-  // write code here
+  constructor() {
+    this.list = new Map();
+    this.uniqueId = 0;
```

Comment:
```suggestion
    this.id = 0;
```
id не может быть не уникальным)